### DISCUSSION TITLE
Pushed Changes Not showing on my Website.

### DISCUSSION THREAD

**Discussion by barnent1** [vercel/community#987] [Help] [answered]
When I push a set of changes to the repository / branch linked to my Vercel project. It still continues to show the old website in the preview window, at the Vercel link, and at the domain link.

The log shows successful. The project shows the correct message given when the branch was committed.

If I use the CLI and push directly. It comes in in Preview mode. Everything looks correct but I am unable to deploy it to production because it tells me to push to the repository. 

Any ideas?

**Comment by amyegan** [comment]
Hi @barnent1! There was an incident related to new builds yesterday that could possibly be related. Are you still having this problem?

When deploying via the CLI, you'll need to use `vercel deploy --prod` to initiate a production deployment instead of a preview. Is that the command that throws an error for you? If you can share the exact error message, and the steps which led to it, that will help me replicate the problem and figure out what is going wrong.

It would also be helpful to know the deployment IDs or links to the CLI and git deployments. Those would help find more information in the logs to figure out why your updates are not being displayed for the git deployment.

**Comment by barnent1** [comment]
Hi Amy,

Yes I got the CLI sorted and that works fine for the same exact code set
that is pushed to github.

So it's beyond me why when I do a push it wouldn't deploy to production
properly.

Take a look at lozfm92jm . This was a git push. When you look at the
preview and everything it's not the site that was pushed it's the old
original one.

Thanks,
Glen





On Wed, Oct 26, 2022 at 10:57 AM Amy ***@***.***> wrote:

> Hi @barnent1 <https://github.com/barnent1>! There was an incident related
> to new builds yesterday that could possibly be related. Are you still
> having this problem?
>
> When deploying via the CLI, you'll need to use vercel deploy --prod to
> initiate a production deployment instead of a preview. Is that the command
> that throws an error for you? If you can share the exact error message, and
> the steps which led to it, that will help me replicate the problem and
> figure out what is going wrong.
>
> It would also be helpful to know the deployment IDs or links to the CLI
> and git deployments. Those would help find more information in the logs to
> figure out why your updates are not being displayed for the git deployment.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/orgs/vercel/discussions/987#discussioncomment-3971026>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AAPRKKOXG6JEYD65OXOBM63WFFBETANCNFSM6AAAAAAROYPCNY>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>


-- 
Sincerely,
Glen Barnhardt

**Comment by barnent1** [comment, marked as answer]
Hey Amy,

Yes, that was it! There was a .vercel directory in the repository and I had
done local builds so it had that info. I have added .vercel to gitignore
and removed it from the repo so that won't happen again.

You may close this ticket.

Sincerely,
Glen Barnhardt

On Wed, Oct 26, 2022 at 11:07 AM Glen Barnhardt ***@***.***>
wrote:

> Hi Amy,
>
> Yes I got the CLI sorted and that works fine for the same exact code set
> that is pushed to github.
>
> So it's beyond me why when I do a push it wouldn't deploy to production
> properly.
>
> Take a look at lozfm92jm . This was a git push. When you look at the
> preview and everything it's not the site that was pushed it's the old
> original one.
>
> Thanks,
> Glen
>
>
>
>
>
> On Wed, Oct 26, 2022 at 10:57 AM Amy ***@***.***> wrote:
>
>> Hi @barnent1 <https://github.com/barnent1>! There was an incident
>> related to new builds yesterday that could possibly be related. Are you
>> still having this problem?
>>
>> When deploying via the CLI, you'll need to use vercel deploy --prod to
>> initiate a production deployment instead of a preview. Is that the command
>> that throws an error for you? If you can share the exact error message, and
>> the steps which led to it, that will help me replicate the problem and
>> figure out what is going wrong.
>>
>> It would also be helpful to know the deployment IDs or links to the CLI
>> and git deployments. Those would help find more information in the logs to
>> figure out why your updates are not being displayed for the git deployment.
>>
>> —
>> Reply to this email directly, view it on GitHub
>> <https://github.com/orgs/vercel/discussions/987#discussioncomment-3971026>,
>> or unsubscribe
>> <https://github.com/notifications/unsubscribe-auth/AAPRKKOXG6JEYD65OXOBM63WFFBETANCNFSM6AAAAAAROYPCNY>
>> .
>> You are receiving this because you were mentioned.Message ID:
>> ***@***.***>
>>
>
>
> --
> Sincerely,
> Glen Barnhardt
>


-- 
Sincerely,
Glen Barnhardt

**Reply by amyegan** [reply]
I'm glad you got it working! Thanks for sharing your solution 😄

**Comment by barnent1** [comment]
No problem!

On Thu, Oct 27, 2022 at 10:00 AM Amy ***@***.***> wrote:

> I'm glad you got it working! Thanks for sharing your solution 😄
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/orgs/vercel/discussions/987#discussioncomment-3982421>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AAPRKKKASUNPZFNS456TSTTWFKDIPANCNFSM6AAAAAAROYPCNY>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>


-- 
Sincerely,
Glen Barnhardt
